lib/game.rb

```diff
@@ -0,0 +1,93 @@
+class Game
+  extend Players
+  include Players
+
+  WIN_COMBINATIONS = [
+    [0,1,2], # Top row
+    [3,4,5], # Middle row
+    [6,7,8], # Bottom row
+    [0,4,8], # Diagonal top left to bottom right
+    [6,4,2], # Diagonal bottom left to top right
+    [0,3,6], # Left vertical
+    [1,4,7], # Middle vertical
+    [2,5,8], # Right vertical
+  ]
+
+  attr_accessor :player_1, :player_2, :board
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new) 
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    board.turn_count.even? ? @player_1 : @player_2
+  end
+
+  def over?
+    draw? || won?
+  end
+
+  def won?
+    WIN_COMBINATIONS.detect do |combination|
+      combo_chars = combination.map { |i| board.cells[i] }
+      combo_chars.all? { |c| c == "X" } || combo_chars.all? { |c| c == "O" }
+    end
+  end
+
+  def draw?
+    !won? && board.full?
+  end
+
+  def winner
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.